Storage_DLL/BaseProduct.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Storage_DLL
+{
+    public class BaseProduct
+    {
+        public ProductTypes ProductType { get; set; }
+        public int DaysToExpiration { get; set; }
+
+        public BaseProduct(ProductTypes productType)
+        {
+            this.ProductType = productType;
+
+            switch (ProductType)
+            {
+                case ProductTypes.Burger:
```

Comment:
Try to substitute switch case with Dictionary and add there product type as a key and expiration date  as a value
Current approach spoils SOLID principle (you will learn it later)